doc/src/base/base.md

```diff
@@ -50,9 +50,11 @@ The following two-word sequences are reserved:
 However, you can create variables with names:
 `abstract`, `mutable`, `primitive` and `type`.
 
-Finally, `where` is parsed as an infix operator for writing parametric method
-and type definitions. Also `in` and `isa` are parsed as infix operators.
-Creation of a variable named `where`, `in` or `isa` is allowed though.
+Finally:
+`outer` is parsed as a keyword when used to modify the scope of a `for` loop variable;
```

Comment:
```suggestion
`outer` is parsed as a keyword when used to modify the scope of a variable in an iteration specification of a `for` loop or `generator` expression;
```
I think it would also make more sense to mention `outer` after `where`, `in` and `isa`, since at least to me, it seems a lot more exotic than those.

Reply:
Sure!